app/frontend/components/domains/home/review-manager/configuration-management-screen/feature-access-screen/inbox-feature-access.tsx

```diff
@@ -1,23 +1,69 @@
-import { observer } from "mobx-react-lite"
-import React from "react"
+import { Button, Container, Flex, Heading, Text, VStack } from "@chakra-ui/react"
+import { CaretLeft } from "@phosphor-icons/react"
+import React, { useState } from "react"
+import { Trans, useTranslation } from "react-i18next"
+import { useNavigate } from "react-router-dom"
 import { useJurisdiction } from "../../../../../../hooks/resources/use-jurisdiction"
-import { FeatureToggleScreen } from "../../../../../shared/base/feature-toggle-screen"
+import { SwitchButton } from "../../../../../shared/buttons/switch-button"
+import { RouterLinkButton } from "../../../../../shared/navigation/router-link-button"
+import { SubmissionsInboxSetupScreen } from "../submissions-inbox-setup-screen"
 
-export const InboxFeatureAccessScreen = observer(() => {
+interface InboxFeatureAccessScreenProps {
+  editPageUrl: string
+}
+
+export const InboxFeatureAccessScreen: React.FC<InboxFeatureAccessScreenProps> = ({ editPageUrl }) => {
   const i18nPrefix = "home.configurationManagement.featureAccess"
+  const { t } = useTranslation()
+  const navigate = useNavigate()
   const { currentJurisdiction } = useJurisdiction()
+  const [isEnabled, setIsEnabled] = useState(currentJurisdiction?.inboxEnabled ?? false)
 
   const handleToggle = (checked) => {
+    setIsEnabled(checked)
     currentJurisdiction.update({ inboxEnabled: checked })
   }
 
   return (
-    <FeatureToggleScreen
-      i18nPrefix={i18nPrefix}
-      featureKey="submissionInbox"
-      backUrl={`/jurisdictions/${currentJurisdiction?.slug}/configuration-management/feature-access/`}
-      isEnabled={currentJurisdiction?.inboxEnabled || false}
-      onToggle={handleToggle}
-    />
+    <Container maxW="container.lg" p={8} as={"main"}>
+      <VStack alignItems={"flex-start"} w={"full"} h={"full"} gap={6}>
+        <Button variant="link" onClick={() => navigate(-1)} leftIcon={<CaretLeft size={20} />} textDecoration="none">
+          {t("ui.back")}
+        </Button>
+        <Flex align="center" w="100%" direction="column" alignItems="flex-start">
+          <Heading as="h1" m={0}>
+            {/* ts-ignore */}
+            {t(`${i18nPrefix}.submissionInbox`)}
+          </Heading>
+          <Text color="text.secondary" fontSize="lg" mt={2}>
+            <Trans
+              i18nKey={`${i18nPrefix}.submissionInboxDescription`}
+              components={{
+                1: (
+                  <RouterLinkButton
+                    variant={"link"}
+                    to={editPageUrl}
+                    fontSize="lg"
+                    textDecoration="none"
+                  ></RouterLinkButton>
+                ),
+              }}
+            />
+          </Text>
+        </Flex>
+      </VStack>
+      <>
+        <SubmissionsInboxSetupScreen />
```

Comment:
You're putting a screen inside of a screen, using nested containers. This is not a great pattern, can you please move the code from inside the SubmissionsInboxSetupScreen to this file? Don't bring in the Container component.